docker/push.sh

```
@@ -1 +1,3 @@
-docker push localhost:5000/comify:0.1.0
\ No newline at end of file
+VERSION=$(jq -r '.version' package.json)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)